src/newSeed/stage2.html

```diff
@@ -61,7 +61,7 @@
       <div class="infoInput logo">
         <div>
           <label for="logo">Project Logo</label>
-          <input id="logo" type="text" placeholder="Valid Uri to project's logo"
+          <input id="logo" type="text" placeholder="Please enter a valid URL to project's logo"
```

Comment:
I don't agree with this change.  Placeholders are always rounds, not requests.  The wording you are proposing is appropriate for validation messages.

A better wording: "Url to the project's logo"  (note that Url is more familiar to the public than Uri)

Reply:
Okay, I will do that. @BartuCulha please note this change is different from what you requested in the cards